### PR TITLE
gui cont tabs chamber: allow overridding the order of mirror axes when parking

### PR DESCRIPTION
**Pull request by pieleric**

### CODE REVIEW

File: src/odemis/gui/cont/tabs.py [hunk 1]
```diff
@@ -1352,19 +1352,24 @@ def _on_switch_btn(self, evt):
         # Note: s axis can only be moved when near engaged pos. So:
         #  * when parking/referencing => move s first, then l
         #  * when engaging => move l first, then s
+        # Some systems are special, default is overridden with MD_AXES_ORDER_REF
         mirror = self.tab_data_model.main.mirror
+        axes_order = mirror.getMetadata().get(model.MD_AXES_ORDER_REF, ("s", "l"))
+        if set(axes_order) != {"s", "l"}:
+            logging.warning("Axes order of mirror is %s, while should have s and l axes",
```
Comment (Sabrina85) [inline src/odemis/gui/cont/tabs.py]: while it should have order s and then l?

File: src/odemis/gui/cont/tabs.py [hunk 2]
```diff
@@ -1352,19 +1352,24 @@ def _on_switch_btn(self, evt):
...
+                            axes_order)
         mstate = self._get_mirror_state(mirror)
+
+        moves = []
         if mstate == MIRROR_PARKED:
             # => Engage
-            f1 = (mirror.moveAbs, {"l": self._pos_engaged["l"]})
-            f2 = (mirror.moveAbs, {"s": self._pos_engaged["s"]})
-            moves = (f1, f2)
+            for a in reversed(axes_order):
```
Comment (Sabrina85) [inline src/odemis/gui/cont/tabs.py]: why reversed?

Comment (pieleric) [reply src/odemis/gui/cont/tabs.py]: If you read comment at l. 1352, it's explained.

File: src/odemis/gui/cont/tabs.py (hunk 1 above)
Comment (pieleric) [reply src/odemis/gui/cont/tabs.py]: No, here the order doesn't matter. What I'm checking is that the user didn't enter weird axes names like `"s "`, or `"L"`.

**Pull request merged**